beacon-chain/rpc/validator_server.go

```diff
@@ -226,3 +218,23 @@ func (vs *ValidatorServer) retrieveActiveValidator(beaconState *pbp2p.BeaconStat
 	}
 	return beaconState.ValidatorRegistry[validatorIdx], nil
 }
+
+func (vs *ValidatorServer) retrieveActiveValidatorsPublicKeys(pubkeys [][]byte) ([][]byte, error) {
```

Comment:
Go method naming preference is to avoid "get" "retrieve" "fetch" verbs, simply call this `activePublicKeys(...) ...`

Reply:
thanks